KE-Apache-Beam-Analysis/MQTT_Publisher/testmachine.go

```diff
@@ -0,0 +1,87 @@
+package main
+
+import (
+	"fmt"
+	"time"
+	"encoding/json"
+	"github.com/yosssi/gmq/mqtt"
```

Comment:
Go, Vendor and KubeEdge packages should be grouped and seperated. Please see our source code files.
Applicable to all files.